packages/grpc-js/src/resolver-dns.ts

```diff
@@ -119,6 +122,18 @@ class DnsResolver implements Resolver {
       details: `Name resolution failed for target ${uriToString(this.target)}`,
       metadata: new Metadata(),
     };
+    
+    const backoffOptions: BackoffOptions = {
+      initialDelay: channelOptions['grpc.initial_reconnect_backoff_ms'],
```

Comment:
what if the key doesn't exist? is it oK?

Reply:
Then the value will be `undefined`, so the `BackoffTimeout` will fall back to the default value, which is 1000 ms.